imagekit-sdk/src/main/java/io/imagekit/sdk/tasks/UrlGen.java

```diff
@@ -29,7 +32,12 @@ public static String getUrl(Map<String,Object> options){
         if (null!=options.get("expireSeconds")) {
             expire = (long) (int)options.get("expireSeconds");
         }
-        return UrlGen.generateUrl(path,urlEndpoint,src,queryParams,transformation,transformationPosition,signed,expire);
+        try {
+            return UrlGen.generateUrl(path,urlEndpoint,src,queryParams,transformation,transformationPosition,signed,expire);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
```

Comment:
why are you doing this? At a minimum, assuming it is correct, I would expect a comment.